Add `4 Digit osu!mania World Cup 2024: Registrations Now Open!` news and wiki post (#10522)
ppy/osu-wiki: news/2023/2023-11-24-4dmwc-registrations-open.md

---
layout: post
title: "4 Digit osu!mania World Cup 2024: Registrations Now Open!"
date: 2023-11-24 15:00:00 +0000
---

> The 5th edition of the 4 Digit osu!mania World Cup opens up. Because key-smashing tournaments shouldn't be exclusive to the upper echelon of players!

Reviewer: ```suggestion
The 5th edition of the 4 Digit osu!mania World Cup opens up, because key-smashing tournaments shouldn't be exclusive to the upper echelon of players!
```

![](/wiki/Tournaments/4DM/2024/img/4dm2024-banner.jpg)

With the [osu!mania World Cup 2023](/wiki/Tournaments/MWC/2023_4K) now behind us, it's time for the up-and-coming rising stars all across the globe to shine! We encourage registering on the most ambitious iteration of the tournament ever conceived. And just like the rosters change with each year, what are the changes we're expected to see on the tournament this time around?

<div align="center">
    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/oF8H1UmV0dI" frameborder="0" allowfullscreen></iframe>
</div>

## What's new

> The pooling and mapping team have been working hard to bring the largest selection of custom maps the tournament has **ever** had! Moreover, the pool structure has deviated a bit from the norm. By not tightly categorizing maps, the aim is to foster a more competitive selection of charts while giving mappers more creative freedom over their works — a win-win situation!

Reviewer: ```suggestion
The pooling and mapping team have been working hard to bring the largest selection of custom maps the tournament has **ever** had! Moreover, the pool structure has deviated a bit from the norm. By not tightly categorising maps, the aim is to foster a more competitive selection of charts while giving mappers more creative freedom over their works — a win-win situation!
```

![](/wiki/Tournaments/4DM/2024/img/4dm2024-pool.jpg)

> The tournament is also introducing a brand-new **catch-up mechanic**. To summarize, if a team reached half the amount of points in any given match, the opponent *must* ban an additional map from the pool not previously protected.

Reviewer: ```suggestion
The tournament is also introducing a brand-new **catch-up mechanic**. To summarise, if a team reached half the amount of points in any given match, the opponent *must* ban an additional map from the pool not previously protected.
```

> Last but not least is the introduction of a new **Qualifier format**! When teams are choosing their lobbies, they are now expected to choose the order in which they'll play through the Qualifier pool. Any order is valid as long as each stage is played at least twice.

Reviewer: ```suggestion
Last but not least is the introduction of a **new Qualifier format**! When teams are choosing their lobbies, they are now expected to choose the order in which they'll play through the Qualifier pool. Any order is valid as long as each stage is played at least twice.
```

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration phase | Nov 24 – Dec 16 (23:59 UTC) |
| Qualifier showcase | Jan 6 |
| Qualifier stage | Jan 13 & Jan 14 |
| Round of 32 | Jan 20 & Jan 21 |
| Round of 16 | Jan 27 & Jan 28 |
| Quarterfinals | Feb 3 & Feb 4 |
| Semifinals | Feb 10 & Feb 11 |
| Finals | Feb 17 & Feb 18 |
| Grand Finals | Feb 24 & Feb 25 |

## Prizes

- **![Gold crown](/wiki/shared/crown-gold.png "1st place") 1st:** Unique profile badge, exclusive banner, 48% of the total prize pool.
- **![Silver crown](/wiki/shared/crown-silver.png "2nd place") 2nd:** Exclusive banner, 36% of the total prize pool.
- **![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") 3rd:** Exclusive banner, 18% of the total prize pool.

## How do I register?

To register, fill out the [following form](http://4dm-registrations.dannypx.com/). Before doing so, please read and understand the following rules:

> - **You must be within the 4 key rank range of #1,000-9,999** both when you submit your registration and until the end of the registration phase.

Reviewer: ```suggestion
- **You must be within the 4K rank range of #1,000-9,999** both when you submit your registration and until the end of the registration phase.
```

- **You must not have violated the [osu! community rules](/wiki/Rules) in the past 12 months.**
- **You must not exceed the global osu!mania rank of #250.**

> - **You must not have been a part of a former winning 4 Digit MWC team.**

Reviewer: ```suggestion
- **You must not have been a part of a former winning 4DM team.**
```

> - **Join the official [Discord Server](https://discord.gg/W2MQ647) for 4DM2024.**

Reviewer: ```suggestion
- **Join the official [Discord server](https://discord.gg/W2MQ647) for 4DM2024.**
```

On top of that, there will be a separate skill-banning phase that will bar players above a certain skill level from participating based on a purely mathematical procedure. Stay tuned in the server to know whether this will affect you or your team!

> To keep up with the tournament's latest announcements, we encourage joining the [4DM Discord Server](https://discord.gg/W2MQ647), checking the [forum post](https://osu.ppy.sh/community/forums/topics/1848473), and reading the [wiki article](/wiki/Tournaments/4DM/2024) for more information. We're eager to see you all there!

Reviewer: ```suggestion
To keep up with the tournament's latest announcements, we encourage joining the [4DM Discord server](https://discord.gg/W2MQ647), checking the [forum post](https://osu.ppy.sh/community/forums/topics/1848473), and reading the [wiki article](/wiki/Tournaments/4DM/2024) for more information. We're eager to see you all there!
```
